index.html

```diff
@@ -1,76 +1,200 @@
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/services/keycloak/" class="btn btn-primary">Learn about <strong>Identity Management</strong></a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+          <a href="/services/crc/"><i class="fa fa-plug"></i> CRC</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
+        </p>
+        <p>
+          <a href="/services/crc/" class="btn btn-primary">Learn about <strong>CRC</strong></a>
+        </p>
+      </div>
+
+      <div class="col-md-4">
+        <h3>
+            <a href="#">Learn More</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
+        <p>
+          To learn more about all of the AeroGear services, read our docs!
+        </p>
+        <p>
+          <a href="#" class="btn btn-primary">Learn More</strong></a>
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
 
-    <section class="custommers">
-      <div class="container">
-        <h2 class="section-header">Mobile development begins here</h2>
-        <p>Using mobile technologies will never be so easy as in the cloud. AeroGear supports several cloud-enabled providers and technologies.</p>
+<section class="sdks">
+  <div class="container">
+    <h2 class="section-header">AeroGear SDKs</h2>
+
+    <div class="row">
+      <div class="col-md-3">
+        <h3>
+          <a href="/sdks/android/"><i class="fa fa-android"></i> Android SDK</a>
+        </h3>
+        <p class="alt">
+          Maecenas ut magna aliquet, facilisis odio ut, sollicitudin nunc. 
+        </p>
         <p>
-          <a href="http://www.feedhenry.com/feedhenry-and-red-hat-pushing-ahead-with-integrations/"><img src="{{ site.baseurl }}/img/logos_home/feedhenry.gif" alt="feedhenry"  /></a>
-          <a href="https://developers.openshift.com/en/xpaas-unified-push.html"><img src="{{ site.baseurl }}/img/logos_home/openshift.gif" alt="openshift"  /></a>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit.
         </p>
-      </div><!-- container -->
-    </section>
\ No newline at end of file
+        <p>
+          <a href="/sdks/android/" class="btn btn-primary">Learn More</a>
+        </p>
+      </div>
+
+      <div class="col-md-3">
+        <h3>
+          <a href="/sdks/cordova/"><i><img class="sdk-icon" src="img/logos_home/cordova-icon.png"></i> Cordova SDK</a>
```

Comment:
Small consistency thing, icon colour for these image icons are different. I _think_ @sedroche may have a custom font icon with Xamarin & Cordova icons lying around. If not, I have one somewhere for some old studio stuff.